src/Application/Services/InitializationService.cs

```diff
@@ -0,0 +1,45 @@
+using System.Linq;
```

Comment:
How about AdminService rather than InitalizationService? The methods don'l really relay the fact that its purpose is to initialize.

Reply:
I took the extra step with a service to have a collection point for all the initialization stuff. Currently, there is only the administrator stuff but I imagine there would be a lot more in the future, all sort of configuration that has to be done when the application is launched for the first time.

If you ever went though a Wordpress installation, that's what this if for. Stuff like:

  - Configuring the site name.
  - Choosing the subdomain or URL schema.
  - Configuring search engine stuff like `robots.txt`.
  - Adding API keys, e.g. Google Analytics.
  - etc.

I think most of this shouldn't be MVP but if I added the admin stuff there because having it hard coded would be dangerous if we don't remove it properly or if it's retained in the database for some reason.

I can name it `AdminService` or `AdministratorService` but that would likely be renamed. On the other hand, it would have to be rewritten anyways.

Do you think I should rename it?

Reply:
I decided to remove this entirely. This first-use page is something that should not go into the minimum viable product.

I re-added something similar to the original code that just creates an 'admin@codidact' user. This will be removed as soon as we use a non in-memory database.